data/xml/2020.wmt.xml

```diff
@@ -878,8 +878,8 @@
     <paper id="73">
       <title>Mitigating Gender Bias in Machine Translation with Target Gender Annotations</title>
       <author><first>Artūrs</first><last>Stafanovičs</last></author>
-      <author><first>Mārcis</first><last>Pinnis</last></author>
       <author><first>Toms</first><last>Bergmanis</last></author>
+      <author><first>Mārcis</first><last>Pinnis</last></author>
       <pages>629–638</pages>
       <abstract>When translating The secretary asked for details. to a language with grammatical gender, it might be necessary to determine the gender of the subject secretary. If the sentence does not contain the necessary information, it is not always possible to disambiguate. In such cases, machine translation systems select the most common translation option, which often corresponds to the stereotypical translations, thus potentially exacerbating prejudice and marginalisation of certain groups and people. We argue that the information necessary for an adequate translation can not always be deduced from the sentence being translated or even might depend on external knowledge. Therefore, in this work, we propose to decouple the task of acquiring the necessary information from the task of learning to translate correctly when such information is available. To that end, we present a method for training machine translation systems to use word-level annotations containing information about subject’s gender. To prepare training data, we annotate regular source language words with grammatical gender information of the corresponding target language words. Using such data to train machine translation systems reduces their reliance on gender stereotypes when information about the subject’s gender is available. Our experiments on five language pairs show that this allows improving accuracy on the WinoMT test set by up to 25.8 percentage points.</abstract>
       <url hash="d841f0d6">2020.wmt-1.73</url>
```

Comment:
Just noticing this here: Can you add (smart) double quotes around "The secretary asked for details." and "secretary"? They are missing from the abstract.